platform/ios/PolyPodApp/FeatureContainer/FeatureContainerView.swift

```diff
@@ -141,7 +141,7 @@ class FeatureWebView: WKWebView {
         openUrlHandler: @escaping (String) -> Void,
         pickFileHandler: @escaping (String?, @escaping (ExternalFile?) -> Void) -> Void
     ) {
-        FeatureStorage.shared.activeFeature = feature
+        PodApi.shared.polyOut.activeFeature = feature
```

Comment:
Seems a bit odd to me to store it in such an overly specific place, rather than `FeatureStorage`. I get that `FeatureStorage` isn't a singleton anymore, but you could just add a `static` to it, hm? `FeatureStorage.activeFeature`. It's not particularly beautiful, and I guess one could make a case for having it only where it's being used for now, but I'd prefer that. Up to you though.

Reply:
`activeFeature` is only used in `PolyOut`, thus I've moved it where it is used. Also, this `activeFeature` has very little to do with `FeatureStorage`. At least for me it makes sense to have `activeFeature` in `PolyOut`, unless it was originally designed with another purpose.